### DISCUSSION TITLE
umi config document: I think some wrong in configuration. Correct me if I am wrong

### DISCUSSION THREAD

**Discussion by tranquanglk94** [umijs/umi#7688] [Q&A]
![image](https://user-images.githubusercontent.com/29516824/145951804-b92d6f68-aa9d-4fe0-bfb2-3ea674efe79d.png)

I think **If UMI_ENV=cloud is specified, the final configuration we get will be:**  the result should be:
{
  a: 1,
  b: 'cloud',
  c: 'cloud',
}
Could anyone correct me this point? Thanks

**Comment by knota-lab** [comment]
![image](https://user-images.githubusercontent.com/26835087/146123455-8ce26835-733f-4a8a-8708-3bba9cb2efe2.png)
